src/components/cylc/Task.vue

```diff
@@ -331,6 +388,9 @@ export default {
     if (context.props.isHeld) {
       taskIconSvgCssClasses.push('held')
     }
+    if (context.props.isQueued) {
```

Comment:
We can only display one indicator at a time (so held or queued).

I think we agreed that held takes precedence? So this should be

```suggestion
    else if (context.props.isQueued) {
```

Reply:
> We can only display one indicator at a time (so held or queued).

We can display more than one if we want to, obviously (e.g. my screenshot) ... but maybe we don't want to...

>  I think we agreed that held takes precedence? 

Yeah, I had forgotten that.  I guess if a task is held it doesn't matter if it's queued or not (queuing is essentially an automated means of holding a task, after all).  So OK, I'll tweak the code to do this tomorrow (have to reposition the held indicator as well, back to the left side). :+1: